tools/assetpipelineeditor/mainwindow.cpp

```diff
@@ -164,6 +173,10 @@ void MainWindow::updateScene(Kuesa::SceneEntity *entity)
         model->update(entity);
     m_ui->collectionBrowser->expandAll();
     m_ui->actionReload->setEnabled(true);
+#if defined(KUESA_DRACO_COMPRESSION)
+    m_ui->actionExport->setEnabled(true);
```

Comment:
I'd go as far as completely hiding it from the GUI for now as long as it's useful only in the draco case. Will change once textures can be marked for post-processing too.

Reply:
done